packages/ui/src/memberships/modals/SwitchMemberModal/SwitchMemberModal.tsx

```diff
@@ -53,12 +56,26 @@ export const SwitchMemberModal = () => {
       </SwitchModalBody>
       {!modalData?.noCreateButton && (
         <SwitchModalFooter>
-          <AddMembershipButtonSwitch
+          <DetailsButton
+            icon={<MyProfileIcon />}
             onClick={() => {
               hideModal()
               showModal<BuyMembershipModalCall>({ modal: 'BuyMembership' })
             }}
+            subtitleText="New Member"
+            titleText="Create a New Membership"
           />
+          {active ? (
+            <DetailsButton
+              icon={<SignOutIcon />}
+              onClick={() => {
+                hideModal()
+                showModal<SignOutModalCall>({ modal: 'SignOut' })
+              }}
+              subtitleText="Sign Out"
+              titleText="Sign out of the active Membership"
```

Comment:
Looks good but `subtitleText` and `titleText` are inverted:
```suggestion
            titleTextNew Member"
            subtitleText="="Create a New Membership"
          />
          {active ? (
            <DetailsButton
              icon={<SignOutIcon />}
              onClick={() => {
                hideModal()
                showModal<SignOutModalCall>({ modal: 'SignOut' })
              }}
              titleText="Sign Out"
              subtitleText="="Sign out of the active Membership"
```